entrypoint.sh

```diff
@@ -1,13 +1,14 @@
 #!/bin/bash
 
+echo "Configuration: MEM=$MEM DISK=$DISK"
+
 #start sshd
 /etc/init.d/ssh start
 
-# Create the ext4 volume image if DISK is set
-if [[ -n "${DISK}" ]] ; then
-    dd if=/dev/zero of=/var/tmp/docker.img bs=1 count=0 seek=${DISK}
-    mkfs.ext4 /var/tmp/docker.img
-fi
+# Create the ext4 volume image for /var/lib/docker
```

Comment:
The image should only be created if it doesn't exist. Otherwise the preserving wouldn't make much sense.

Reply:
fixed